test/utils/RasTestHelper.scala

```diff
@@ -30,7 +29,7 @@ import play.api.i18n
 import play.api.i18n.{MessagesApi, MessagesImpl}
 import play.api.mvc._
 import play.api.test.FakeRequest
-import play.api.test.Helpers.{contentAsString, stubBodyParser, stubControllerComponents, stubMessagesApi}
+import play.api.test.Helpers.{contentAsString, stubBodyParser, stubControllerComponents, stubMessagesApi, _}
```

Comment:
If we need to import everything does it not make more sense to just have the `_` rather than the 4 other things and the `_`